src/components/Text.tsx

```diff
@@ -1,8 +1,33 @@
 import { styled } from "@storybook/theming";
 
-export const Text = styled.div<{ small?: boolean }>(({ small }) => ({
-  fontSize: small ? 12 : 14,
-  lineHeight: small ? "18px" : "20px",
-  textAlign: "center",
-  textWrap: "balance",
-}));
+export const Text = styled.div<{ center?: boolean; muted?: boolean; small?: boolean }>(
+  ({ center, small, theme }) => ({
+    display: "inline-block",
+    color: theme.color.defaultText,
+    fontSize: small ? theme.typography.size.s1 : theme.typography.size.s2 - 1,
```

Comment:
Storybook changed the base font sizes to 12px and 14px. Likewise, this should be 12px or 14px. There are a handful of exceptions like Button. Toolbar buttons should still be 12px, though.